### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by lreading** [lreading/owasp-td-jekyll#38]
Accessibility should be a first class feature, especially for documentation. 
As a starting point, we could add an a11y check as part of the PR and release process.
